aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -303,6 +303,11 @@ Tensor& mvlgamma_(Tensor& self, int64_t p) {
   IMPLEMENT_UNARY_OP_OUT_INPLACE(op, cpu, CPU)                         \
   IMPLEMENT_UNARY_OP_OUT_INPLACE(op, cuda, CUDA)
 
+#define IMPLEMENT_UNARY_OP_VEC_HB(op)                                \
```

Comment:
this piece of code looks very familiar ... #74 ?

Reply:
> this piece of code looks very familiar ... #74 ?

Yes, I accidentally included this as well. I will remove it.